### PR TITLE
Setup React Frontend

### PR DESCRIPTION
**Pull request by james-kaguru**

### CODE REVIEW

File: frontend/src/pages/Home.tsx [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+//   return (
+//     <nav className={"p-3 flex flex-row items-center justify-start"}>
+//       <a href="/" className={"text-primary border-b-2 border-primary p-2"}>
+//         Home
+//       </a>
+//     </nav>
+//   );
+// }
+
+function Sponsors() {
```
Comment (spankie) [inline frontend/src/pages/Home.tsx:18]: can we have a components folder and move these components in there so they can be reused in the other pages?

**Comment by nutcas3** [comment]
Hello it looks good but can u take a screenshot of it running on the browser kindly

**Review by spankie — CHANGES_REQUESTED** [review]

**Comment by nutcas3** [comment]
Hello 
Can u show how but runs in the browser and  write how u can run both apps in the readme

**Comment by james-kaguru** [comment]
@nutcas3 here is the screenshot
localhost_5173_(desktop)

File: frontend/src/pages/Home.tsx (hunk 1 above)
Comment (james-kaguru) [reply frontend/src/pages/Home.tsx:18]: alright @spankie lemmi work that

**Comment by james-kaguru** [comment]
> Hello Can u show how but runs in the browser and write how u can run both apps in the readme

okay

**Comment by nutcas3** [comment]
let me run and merge it
lgtm

**Comment by james-kaguru** [comment]
@nutcas3  Ive left our instructions for the backend since its not yet been worked on

File: frontend/src/pages/Home.tsx (hunk 1 above)
Comment (james-kaguru) [reply frontend/src/pages/Home.tsx]: @nutcas3 Ive already pushed the changes inline with the request..should I revert them?

**Comment by nutcas3** [comment]
Kindly move the other issues to a new PR,
thanks

**Pull request merged**